utils/index.js

```diff
@@ -194,6 +194,17 @@ const doYesNo = (claim, fields) => {
   }
 }
 
+
+const postAmount = (amount, locale) => {
+  if(amount && (locale === 'fr')) {
+    const formattedAmount = amount.replace(',','.').replace(' ', '')
```

Comment:
It's a bit pedantic, but can we put a space between the args the first replace, just so it's a bit easier to read?

```
amount.replace(',', '.').replace(' ', '')
```

Reply:
Also, this will only catch the first instance of a thing unless you make it a global replace.

```
"1 1 1 1 1 1 1".replace(' ', '') 
// results in "11 1 1 1 1 1"
```